### DISCUSSION TITLE
Ripetere automazione

### DISCUSSION THREAD

**Discussion by joecas65** [indomus/forum#2288] [HUB: Home Assistant]
Salve a tutti.
Ho creato un'automazione che mi notifica se la finestra del bagno rimane aperta per più di mezzora.
La cosa funziona bene, ma ricevo solo una notifica dopo appunto 30 minuti.
Quello che vorrei è ricevere una notifica ogni mezzora se la finestra non viene chiusa.
Spero di essermi spiegato.
grazie

**Comment by Scialla** [comment]
Guardati il while loop o il repeat until
https://www.home-assistant.io/docs/scripts/#while-loop

**Reply by joecas65** [reply]
Grazie ci provo...
